api/src/main/resources/hibernate.cfg.xml

```diff
@@ -57,7 +57,6 @@
 		<mapping resource="org/openmrs/api/db/hibernate/Person.hbm.xml" />
 		<mapping resource="org/openmrs/api/db/hibernate/PersonAttribute.hbm.xml" />
 		<mapping resource="org/openmrs/api/db/hibernate/PersonAttributeType.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/PersonAddress.hbm.xml" />
 		<mapping resource="org/openmrs/api/db/hibernate/PersonMergeLog.hbm.xml" />		
```

Comment:
you have removed this file from the hibernate.cfg.xml but havent deleted the file itself, it should be removed since it wont be used anymore.